src/locales/ca_ES/achv.ts

```diff
@@ -271,6 +271,10 @@ export const PGMachv: AchievementTranslationEntries = {
   "FRESH_START": {
     name: "First Try!",
     description: "Complete the Fresh Start challenge."
+  },
+  "INVERSE_BATTLE": {
+    name: "Mirror rorriM",
+    description: "Complete the Inverse Battle challenge.\n.egnellahc elttaB esrevnI eht etelpmoC"
```

Comment:
```suggestion
    description: "Complete the Inverse Battle challenge.\n.ǝƃuǝllɐɥɔ ǝlʇʇɐq ǝsɹǝʌuI ǝɥʇ ǝʇǝldɯoƆ"
```
Idk if the font can render that :D

Reply:
do you want to change english translation?
( there is no ca_ES translation anymore? )